.travis.yml

```diff
@@ -3,7 +3,7 @@ language: node_js
 sudo: required
 
```

Comment:
I thought we removed Travis completely. How come we still have a .travis.yml?

Reply:
Not sure but I can remove it as part of this PR.